.doc_gen/metadata/sts_metadata.yaml

```diff
@@ -131,8 +136,12 @@ sts_Scenario_ConstructFederatedUrl:
 sts_Scenario_SessionTokenMfa:
   title: Get a session token that requires an MFA token with &STS; using an &AWS; SDK
   title_abbrev: Get a session token that requires an MFA token
-  synopsis: get a session token that requires a mulfi-factor authentication
-    (MFA) token with &STS; and use  temporary credentials to access an &AWS; resource.
+  synopsis_list:
+    - Create an &IAM; role that grants permission to list &S3; buckets.
```

Comment:
&S3; --> S3

Reply:
This would be first mention in this example so I think we keep this one as &S3;.